webclient/services/mailer.py

```diff
@@ -698,7 +715,8 @@ def get_en01_data() -> Dict:
             projekt__organizace=F('historie__historie__uzivatel__spoluprace_badatelu__vedouci__organizace'),
             historie__historie__uzivatel__spoluprace_badatelu__vedouci__notification_types__ident_cely='S-E-N-01',
             id__in=Subquery(subquery_sn12),
-            historie__historie__datum_zmeny__date=timezone.now().date() + timedelta(days=-1)
+            historie__historie__datum_zmeny__gt=timezone.now() + timedelta(days=-2),
+            historie__historie__datum_zmeny__lt=timezone.now() + timedelta(days=-1),
```

Comment:
@pesikj Jestli to dobře chápu, tak zde se to bude řídit konkrétním časem, nikoli obecným datumem. To ale není ideální, protože to může vést k tomu, že se některá oznámení přeskočí (kvůli odlišnému času spuštění apod.). Je třeba podmínku nastavit tak, aby to bylo: `datum.dnes - 2 dny < historie.datum_zmeny < datum.dnes`. Možná to tak ale je, jen to špatně čtu.